zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -180,6 +181,22 @@ class ObjectManagementFacade(
         }
     }
 
+    fun deleteObject(
+        objectName: String,
+        objectId: UUID
+    ): HttpStatus {
+        val accessObject = getAccessObject(objectName)
+
+        try {
+            logger.trace { "Deleting object $objectId" }
+            return accessObject.objectenApiPlugin.deleteObject(
+                URI(accessObject.objectenApiPlugin.url.toString() + "objects/" + objectId)
```

Comment:
Nitpicking, but in kotlin you usually want to use the [string templates](https://kotlinlang.org/docs/basic-syntax.html#string-templates) because it's more efficient.